<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatServer</class>
 <widget class="QMainWindow" name="ChatServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>74</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>431</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="btnStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="btnStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>211</width>
         <height>78</height>
        </size>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>671</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditText"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
